eng/common/templates/jobs/copy-base-images.yml

```
@@ -0,0 +1,28 @@
+parameters:
+  name: null
+  additionalOptions: null
+  
+jobs:
```

Should this be skipped for PR validation because it won't have access to the necessary credentials?

---

I see that the build is attempting to copy the images and fails but the failure is ignored.  I think it would be worthwhile to still execute this in a PR but with a dry-run option instead.  This would be the same approach that is done in the publish stage.  It helps to identify potential issues that could be caught in a PR even if it is a dry run.

---

That makes sense.